shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -691,26 +543,6 @@ export default {
         return day(time).format(this.dateTimeFormatStr);
       }
     },
-
-    machineSortGenerationFn() {
```

Comment:
This appears to only be necessary if we have the fake machines and nodes but neither appear to be necessary.